cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -47,6 +47,8 @@ spec:
   crashCollector:
     disable: false
   storage:
+    # when skipApplyAllPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement.
+    skipApplyAllPlacement: true
```

Comment:
Is this a setting we intend the user to change if we set it to true for the PVC scenario exclusively? If so, when? If can't we internally determine this and merge when necessary?

Reply:
Let's add this at the end of the storage section of the example. Since it's an advanced example I don't think it should be the first setting people see.